project_recalculate/models/project_task.py

```diff
@@ -193,39 +202,71 @@ def _interval_context_tz(self, interval):
         start = fields.Datetime.context_timestamp(self, start)
         end = datetime.now().replace(hour=interval[1])
         end = fields.Datetime.context_timestamp(self, end)
-        return (start.hour, end.hour)
+        result = Intervals()
+        result._items.append((start, end))
```

Comment:
Should you access to the protected variable `_items`? Isn't there a way to add elements directly through a public API?

Reply:
Thinking better I think it does not make much sense to use the _interval_default_get method. Since in v12, the `resource.calendar` methods no longer receive a` default_interval` as a parameter.
So, it is not necessary to create an Intervals object